operator/src/test/java/oracle/kubernetes/weblogic/domain/model/DomainStatusConditionMatcher.java

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) 2019, 2022, Oracle and/or its affiliates.
```

Comment:
Is this a new class? if so, the copyright is not correct.

Reply:
I renamed it. It used to be called DomainConditionMatcher. Apparently git doesn't preserve history in such cases.

Reply:
I see it now.

Reply:
@russgold `git add -A` will allow git to preserve the history across file renames…

Reply:
Ah, thanks. I had thought it would do so automatically. I'll take advantage of this in the future.